portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/CreateABTest.testcase

```diff
@@ -66,7 +63,67 @@ definition {
 	@description = "Validate if the user can create an A/B Test"
 	@priority = "5"
 	test CreateABTestByBounceRate {
-		ABTest.createABTestByBounceRate(testTitle = "Create A/B Test Title");
+		property minimum.slave.ram = "32";
+
+		ABTest.createABTest(
+			goal = "Bounce Rate",
+			testTitle = "AB test title");
+
+		ABTest.createVariant(variantName = "Variant name");
+
+		ABTest.runTest();
+
+		ABTest.checkAnalyticsCloudInfo();
+
+		if (IsElementNotPresent(locator1 = "ABTest#AB_TEST_TITLE_AC_SIDE")) {
+			SelectWindow(locator1 = "title=Analytics Cloud");
+
+			AnalyticsCloud.loginAnalyticsCloud(
+				emailAddress = "test@liferay.com",
+				password = "test");
+		}
+		SelectWindow(locator1 = "title=Tests - Analytics Cloud");
+
```

Comment:
This is what's causing the test failure in **CreateABTest#CreateABTestByBounceRate**: 
* While running the `ABTest.checkAnalyticsCloudInfo` macro, it clicks the `View Data Analytics Cloud` button and the Analytics cloud URL is opened in a new tab.
* From here, I'm not sure if this locator is something on Portal or AC `ABTest#AB_TEST_TITLE_AC_SIDE` 
    * I'm guessing that's on the AC side though, and what I think is happening in CI is that the page is loading slowly. The `IsElementNotPresent` evaluate happens instantaneously and is likely returning true. 
* I think the `SelectWindow` step may not be necessary, as WebDriver automatically switches context to the new opened tab (examining the screenshots). You might not have encountered this locally because the test runs faster locally. Try this locally to confirm if you can remove the `SelectWindow` and then run it in CI. 
    * If the `SelectWindow` is necessary though, try something like: 
```
		SelectWindow(locator1 = "name=undefined"); // Switches to another tab. You don't have to rely on the titles. Poshi will just switch it to the other window that isn't in currently in context which might be helpful as it seems like the title is variable depending on if the user is logged in or not.

		if (IsElementNotPresent(locator1 = "ABTest#AB_TEST_TITLE_AC_SIDE")) {
			AnalyticsCloud.loginAnalyticsCloud(
				emailAddress = "test@liferay.com",
				password = "test");
		}
```

 I believe WebDriver automatically switches context to this new window, so the `SelectWindow` might not be necessary.